src/errors/tests/ActiveNetworkNotFound.test.js

```diff
@@ -0,0 +1,43 @@
+// @flow
```

Comment:
let's move tests to `src/errors`.
now we don't understand how deep can specific `tests` dir could be placed, so it's ok to keep all tests near origin src file

Reply:
A few days ago we discussed that we will use `tests` subfolder for unit tests. 
@stcherenkov @Creepypoke @Dragonesis What do you think about this?